features/font-variant-position.yml

```
@@ -0,0 +1,11 @@
+name: Font variants for position
+description: The `font-variant-position` CSS property sets whether alternate glyphs are used to display text as subscript and superscript.
```

Style nit: "is used to" and "are used to" are often a sign that the verb following can moved for a more active and concise sentence.

```suggestion
description: The `font-variant-position` CSS property sets whether to use alternate glyphs for subscript and superscript text.
```